HW_5_2023/task_26.py

```diff
@@ -0,0 +1,19 @@
+'''Задача 26:  Напишите программу, которая на вход принимает два числа A и B, 
+и возводит число А в целую степень B с помощью рекурсии.
+
+*Пример:*
+
+A = 3; B = 5 -> 243 (3⁵)
+    A = 2; B = 3 -> 8 '''
+def power(A, B):
+    if B == 0:
+        return 1
+    else:
+        return A * power(A, B-1)
```

Comment:
`else` не обязателен, так как `return` завершает работу функции
```suggestion
    return A * power(A, B-1)
```